mcsema/Arch/Arch.cpp

```diff
@@ -331,6 +342,102 @@ size_t ArchDecodeInstruction(const uint8_t *bytes, const uint8_t *bytes_end,
   return total_size;
 }
 
+// Convert the given assembly instruction into an inline ASM operation in lieu
+// of decompiling it.
+void ArchBuildInlineAsm(llvm::MCInst &inst, llvm::BasicBlock *block) {
+  auto opcode = inst.getOpcode();
+
+  // Use the printer to build the ASM string. We'll need to escape the $ in
+  // register names with $$.
+  std::string asmString, constraints;
```

Comment:
What I mean is as a replacement for things like `asmString += "$$";` So instead you'd have ```std::stringstream asmString;
std::strinstream constraints;
...
asmString << "$$";
...
constraints << "{" << gMRI->getName(op.getReg()) << "}";
...
auto callee = llvm::InlineAsm::get(asmTy, asmString.str(), constraints.str(), false);
```